azerothcore/azerothcore-wotlk
# fix(DB/Gameobject): Add node pools to solid chests in Winterspring

## Changes Proposed:
- Added node pool to the chests in General Winterspring (GUID 85800, 87393), 
- Winterfall Village (GUID 85798, 49091, 85799, 85803), 
- Lake Kel'Theril (GUID 85802, 49088), 
- Timbermaw Post (GUID 49089, 85796) 
- and Frostfire Hot Springs (GUID 85801, 49090)

## Issues Addressed:

- Closes #6625

## SOURCE:


## Tests Performed:

Done the deep tests as described in how to test the changes


## How to Test the Changes:


Question: How in-depth should a step-by-step be?

Shallow:
1. Apply SQL
2. Check in game for each zone that only one chest has spawned

Deep:
1. Apply SQL
2. Check that the pool templates are added correctly 
- There should be pool templates with entry 11660, 11661, 11662, 11663 and 11664
- They should all have a limit of 1
- Entry 11660, 11661, 11662, 11663 and 11664 should have descriptions mentioning Winerspring, Winterfall, Lake KelTheril, Timbermaw and Frostfire respectively.
3. Check that each pool in pool_gameobject contains the correct GUIDs, as mentioned in the prior step and changes proposed.
4. Check that those GUIDs are not in any other pools.
5. Check in game for each zone that only one chest has spawned

## Known Issues and TODO List:


- The issue #6625 mentioned random spawn times, this could be an idea to add. I do, however, not know how to implement it.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
